src/gapi_lwt.ml

```diff
@@ -0,0 +1,13 @@
+let promise (f : _ -> unit) =
+  let (promise, resolver) = Lwt.task () in
```

Comment:
```suggestion
  let promise, resolver = Lwt.task () in
```

Reply:
This isn't me that add those parenthesis, their automatically added every time I try to save after delete it on "Emacs".
"ocamlformat" impose it this way it seems...